install/deploy_management_nodes.md

```diff
@@ -347,21 +351,24 @@ be performed are in the [Deploy](#deploy) section.
 
 1. Validate that the LiveCD is ready for installing NCNs.
 
-   > Observe the output of the checks and note any failures, then remediate them.
+   Observe the output of the checks and note any failures, then remediate them.
```

Comment:
This might be outside the scope of the changes for this ticket, but we should provide a pointer to a troubleshooting topic or more information about how to deal with failures.

Reply:
Yeah, I agree that we need to do that. I'm not sure the best place to point them off-hand, so I will open a new ticket for that piece of this.

Reply:
Opened [CASMINST-4688](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4688) for this.